introduction_to_applying_machine_learning/lightgbm_catboost_tabtransformer_autogluon_churn/churn-prediction-lightgbm-catboost-tabtransformer-autogluon.ipynb

```diff
@@ -0,0 +1,1855 @@
+{
```

Comment:
<p>Optionally add a Conclusion section summarizing what the customer accomplished and recommending any next steps or further applications for these techniques.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='107'/>

Reply:
<p>Section 7 is actually my conclusion section, I tell customers "Now you can use this template to evaluate the performance of LightGBM, CatBoost, TabTransformer, and AutoGluon-Tabular on your own dataset."</p>